### DISCUSSION TITLE
How should we organize and maintain this discussion forum?

### DISCUSSION THREAD

**Discussion by lukenoaa** [NOAA-Omics/NOAA-Omics#2] [General]
Questions to consider:

- Which categories do we want to have?
- How often do we need to check the discussion?
- Who will maintain the discussion (rotating schedule)?
- How will content be fed back into other resources (e.g., GitHub issues to DMG repo)?
- Side note: Do we have a critical mass to maintain this discussion forum?

@ksilnoaa @marinednadude

**Comment by marinednadude** [comment]
Suggested categories:
1. eDNA Field
2. eDNA Lab
3. eDNA Bioinformatics
4. Population Genetics
5. Transciptomics
6. Metabolomics
7. Lipidomics
8. Data Management

How often: monthly at minimum. Folks encouraged to look more regularly

NOAA Technical Team can serve as rotating schedule. Job is not to answer every question but to check on questions, and tag people that may know the answer.

Closing the loop is important. Not sure how to best do that. But maybe part of the rotation job? At the very least make sure to bring it up at relevant 'Omics meetings?

I don't know how to answer that question, but there are apparently 80 NOAA 'Omics PIs or so which to me seems like it possibly be a critical mass? If we build it they will come? I think we give it a spin and if sputters out/never gains traction then c'est la vie.

**Reply by ksilnoaa** [reply]
My suggested categories:

1. eDNA Field/Lab Protocols
2. eDNA Bioinformatics
3. PopGen Field/Lab Protocols
4. PopGen Bioinformatics
6. Functional Genomics Field/Lab Protocols
7. Functional Genomics Bioinformatics
8. Metabolomics
9. Data Management
10. General

**Reply by lukenoaa** [reply]
Thanks for the suggestions, @marinednadude and @ksilnoaa! FYI, you can Subscribe to a discussion to get alerts. The button is on the right side of a post and looks like this:
<img width="351" alt="image" src="https://github.com/user-attachments/assets/b74232e3-6b28-48f3-bf0c-570bb54d8d06">

**Comment by lukenoaa** [comment]
At our meeting on Oct 24, 2024, we decided to add the Categories:

- Field
- Lab
- Bioinformatics

**Comment by lukenoaa** [comment]
Here's a video about how to use GitHub Discussions:
https://www.youtube.com/watch?v=FjyPDaVxYRQ

**Comment by sformel-usgs** [comment]
Testing if external folks can participate...

**Reply by sformel-usgs** [reply]
aww yeahhhh we can.

**Reply by lukenoaa** [reply]
And boom goes the dynamite.

**Reply by JesseHA01** [reply]
Congratulations on getting underway.
